new/LIT7083QeneTadewewotu.xml

```diff
@@ -0,0 +1,89 @@
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has eight lines, meaning
+                    that it might be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-01">Created entity together with Nafisa Valieva and Michael Hensley</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7083QeneTadewewotu" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7083QeneTadewewotu" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 22r, 23r. Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">ተጼውዎቱ፡ ለ<choice><sic>ስ</sic><corr resp="JK">ሰ</corr></choice>ብእ፡ እመ፡ 
+                        ከመ፡ ጽዮን፡ ኮነ፡ ምህርካ፡ ጽኑእ፡ እም፡ ተመነነ፤</l>
+                    <l n="2">እመ፡ ጽዮንሃ፡ እለ፡ ጼወው፡ 
+                        ፍጡ<del rend="overUnderlined">ን</del><add rend="above">ነ</add>፡ እስመ፡ 
+                        ኮ<choice><sic>ኦ</sic><corr resp="JK">ኑ</corr></choice>፡ ጺውዋነ፤</l>
+                    <l n="3">ወብሩህ፡ ገ<choice><sic>ጸ</sic><corr resp="JK">ጾ</corr></choice>ሙ፡ መልብስ፡ ኀዘን፡ ተከድነ፤</l>
+                    <l n="4">እስመ፡ ቅድመ፡ አብዝሖ፡ ፍስሐ፡ ያመጽእ፡ ኃዘነ፤</l><pb n="23r"/>
+                    <l n="5">መኳንንተ፡ ሥርአት፡ ወህግ፡ ካህናቲ<choice><sic>፤ሂ</sic><corr resp="JK">ሃ</corr></choice>፡ 
```

Comment:
I think, the ፤ before ሂ was crossed out. You can either delete it, or mark it with `<del>`.

Reply:
You are right: I marked it as part of my own correction, but it had already been cancelled by the scribe